### PR TITLE
Animations

### PR DESCRIPTION
**Pull request by shaileshpatil-work**
This branch contains both CSS as well as Jquery animations with examples.And Recommedations.

### CODE REVIEW

File: Animations/Css Animations/Examples/custom.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+		transform: none;
+	}
+}
+
+/*FadeInRight*/
+
+.fadeInRight {
+	animation:fadeInRight 1s ease-in-out 0.5s 2;
+}
+@keyframes fadeInRight {
```
Comment (hitesh-patil) [inline Animations/Css Animations/Examples/custom.css]: use **@-webkit-** for safari version support (4.0 - 8.0).

Comment (shaileshpatil-work) [reply Animations/Css Animations/Examples/custom.css]: OK

File: Animations/Css Animations/Examples/custom.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+.outerBox {
+	margin-bottom: 20px;
+}
+
+/*fadeInLeft*/
+
+.fadeInLeft {
+	animation:fadeInLeft 1s ease-in-out 0.5s;
+}
+@keyframes fadeInLeft {
```
Comment (abp437) [inline Animations/Css Animations/Examples/custom.css]: Add a line break.

File: Animations/Css Animations/Examples/custom.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+		opacity: 1;
+		transform: none;
+	}
+}
+
+/*FadeInRight*/
+
+.fadeInRight {
+	animation:fadeInRight 1s ease-in-out 0.5s 2;
+}
```
Comment (abp437) [inline Animations/Css Animations/Examples/custom.css:57]: Add a line break.

File: Animations/Css Animations/Examples/custom.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+@keyframes fadeInDown {
+	0% {
+		opacity: 0;
+		transform: translate(0,-100%);
+	}
+	100% {
+		opacity: 1;
+		transform: none;
+	}
+}
```
Comment (abp437) [inline Animations/Css Animations/Examples/custom.css]: Add a line break.

File: Animations/Css Animations/Examples/index.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <title>CSS Animations</title>
```
Comment (abp437) [inline Animations/Css Animations/Examples/index.html]: `<meta>` tags Missing

File: Animations/Css Animations/Examples/index.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+         <h3>Fade In Up</h3>
+         <p class="fadeInUp">Fade In Up</p>
+      </div>
+      <!--fadeInDown-->
+      <div class="OuterBox">
+         <h3>Fade In Up</h3>
+         <p class="fadeInDown">Fade In Up</p>
+      </div>
+   </body>
+</html>
```
Comment (abp437) [inline Animations/Css Animations/Examples/index.html]: Add a line break.

File: Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html [hunk 7]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Animation Chaining</title>
+		<style type="text/css">
```
Comment (abp437) [inline Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html]: Add in a separate CSS file.

File: Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+			p.para
```
Comment (abp437) [inline Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html]: Handle via Classes only.

File: Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+		<p class="animationPara">PARA</p>
+
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+		<script type="text/javascript">
+			$(document).ready(function(){
+			$(".para").animate({'marginLeft':'50px'}).css({'color':'red'}).animate({'marginLeft':'150px'}).css({'color':'green'});
+			});
+		</script>
+	</body>
+</html>
```
Comment (abp437) [inline Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html]: Add a line break.

File: Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css [hunk 10]
```diff
@@ -0,0 +1,61 @@
+* {
+	margin:0;
+	padding: 0;
+	text-decoration: none;
+	list-style-type: none;
+}
+ul li {
```
Comment (abp437) [inline Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css:7]: Indentations missing in the file

File: Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+	}
+	ul li.clear {
+		float: none;
+		margin: 0;
+		clear: both;
+	}
+	#First,#Second,#Third {
+		padding-top: 78px;
+	}
+}
```
Comment (abp437) [inline Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css]: Add a line break.

File: Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+	}
+	ul li {
+		float: left;
+	}
+	ul li.clear {
+		float: none;
+		margin: 0;
+		clear: both;
+	}
+	#First,#Second,#Third {
```
Comment (abp437) [inline Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css]: Why id's and padding-top in css?

File: Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/index.html [hunk 13]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <title></title>
```
Comment (abp437) [inline Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/index.html]: Title Missing

File: Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/index.html [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+      <link rel="stylesheet" type="text/css" href="custom.css">
```
Comment (abp437) [inline Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/index.html]: Meta Tags Missing

File: Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/index.html [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+   </head>
+   <body>
+      <div>
+      <div class="leftSection">
+         <ul>
+            <li><a id="one" href="#First">First</a></li>
+            <li><a id="two" href="#Second">Second</a></li>
+            <li><a id="three" href="#Third">Third</a></li>
+            <li class="clear"></li>
```
Comment (abp437) [inline Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/index.html]: Create and add clearfix class to `<ul>`, remove empty `<li>`

**Review by abp437 — CHANGES_REQUESTED** [review]
Need Changes and I can still spot the id's in urls of the click to scroll page. Please remove that.

File: Animations/Css Animations/Examples/custom.css (hunk 1 above)
Comment (shaileshpatil-work) [reply Animations/Css Animations/Examples/custom.css]: Done

File: Animations/Css Animations/Examples/custom.css (hunk 2 above)
Comment (shaileshpatil-work) [reply Animations/Css Animations/Examples/custom.css]: Done

File: Animations/Css Animations/Examples/custom.css (hunk 3 above)
Comment (shaileshpatil-work) [reply Animations/Css Animations/Examples/custom.css:57]: Done

File: Animations/Css Animations/Examples/custom.css (hunk 4 above)
Comment (shaileshpatil-work) [reply Animations/Css Animations/Examples/custom.css]: Done

File: Animations/Css Animations/Examples/index.html (hunk 5 above)
Comment (shaileshpatil-work) [reply Animations/Css Animations/Examples/index.html]: Done

File: Animations/Css Animations/Examples/index.html (hunk 6 above)
Comment (shaileshpatil-work) [reply Animations/Css Animations/Examples/index.html]: Done

File: Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html (hunk 7 above)
Comment (shaileshpatil-work) [reply Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html]: Done

File: Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html (hunk 8 above)
Comment (shaileshpatil-work) [reply Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html]: Done

File: Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html (hunk 9 above)
Comment (shaileshpatil-work) [reply Animations/jquery Animations/Examples/jquery animation failure compared with css animations.html]: Done

File: Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css (hunk 10 above)
Comment (shaileshpatil-work) [reply Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css:7]: Done

File: Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css (hunk 12 above)
Comment (shaileshpatil-work) [reply Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css]: For mobile view position of menubar fixed so when u click on anchors in menu each div goes to top so padding to each div is given.I don't know its solution it is set by offest .Please give its solution.That coding is faulty as it added more space between divs.

File: Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css (hunk 11 above)
Comment (shaileshpatil-work) [reply Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css]: Done

File: Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/index.html (hunk 14 above)
Comment (shaileshpatil-work) [reply Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/index.html]: Done

File: Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/index.html (hunk 15 above)
Comment (shaileshpatil-work) [reply Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/index.html]: Done

**Review by shaileshpatil-work — COMMENTED** [review]
See upadated cahnges

**Review by abp437 — APPROVED** [review]
I will be making the necessary changes in my PR
